imgui-binding/src/main/java/imgui/extension/imguiknobs/flag/ImGuiKnobFlags.java

```diff
@@ -0,0 +1,15 @@
+package imgui.extension.imguiknobs.flag;
+
+public final class ImGuiKnobFlags {
```

Comment:
May I ask you to convert those flags to the general style? For example: https://github.com/SpaiR/imgui-java/blob/main/imgui-binding/src/main/java/imgui/extension/imguizmo/flag/Operation.java

- Fields in `UPPER_CASE`
- Private constructors without body (no need to throw exception